client/http.go

```diff
@@ -198,6 +175,11 @@ func (h *httpClient) Watch(ctx context.Context) <-chan Result {
 	return pollingWatcher(ctx, h, h.chainInfo, h.l)
 }
 
+// Info returns information about the chain.
+func (h *httpClient) Info(ctx context.Context) (*chain.Info, error) {
+	return h.chainInfo, nil
```

Comment:
Should this call `FetchChainInfo` instead?

Reply:
there's no requirement for it to perform a `Fetch` though? it may have been created with a full info and is just returning it.

Reply:
`FetchChainInfo` does that check and also will get it if not available

Reply:
ah - i misread initially and thought you were wondering if the method should be renamed :)

fetchChainInfo is called in the constructor only, so chainInfo is guaranteed to be set for an http client.